rpcs3/rpcs3_version.cpp

```diff
@@ -2,7 +2,13 @@
 #include "rpcs3_version.h"
 #include "git-version.h"
 
+char const* get_branch()
+{
+	return RPCS3_GIT_BRANCH;
+}
+
 namespace rpcs3
 {
 	const extern utils::version version{ 0, 0, 3, utils::version_type::alpha, 1, RPCS3_GIT_VERSION };
+	const char* branch{get_branch()};
```

Comment:
This will be nullptr at startup and may crash depending on initialization order. It must be a function callable from the outside.

Reply:
Think it should avoid that in the latest commit, not 100% sure though.